test/evergreen.yml

```diff
@@ -504,16 +504,28 @@ functions:
         display_name: WT Hang Analyzer Output - Execution ${execution}
         remote_file: wiredtiger/${build_variant}/${revision}/wt_hang_analyzer/wt-hang-analyzer_${task_name}_${build_id}${postfix|}.tgz
 
-  "print python stacktrace":
-    command: shell.exec
-    params:
-      working_dir: "wiredtiger/cmake_build"
-      script: |
-        set -o errexit
-        set -o verbose
-        if [ -d "WT_TEST" ]; then
-          ${python_binary|python3} ../test/evergreen/print_python_stack_trace.py -e ${python_binary|python3} -c WT_TEST -l .
-        fi
+  "dump stacktraces":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/cmake_build"
+        script: |
+          set -o errexit
+          set -o verbose
+          if [ -d "WT_TEST" ]; then
+            ${python_binary|python3} ../test/evergreen/print_stack_trace.py -e ${python_binary|python3} -c WT_TEST -l .
+          fi
+          ${python_binary|python3} ../test/evergreen/print_stack_trace.py -e ./test/format/t -c test/format -l .
+    - command: s3.put
+      params:
+        aws_secret: ${aws_secret}
+        aws_key: ${aws_key}
+        local_files_include_filter:
+          - wiredtiger/cmake_build/*stacktrace.txt
+          - wiredtiger/cmake_build/test/format/*stacktrace.txt
```

Comment:
I think you don't need this?

Reply:
Yes, I suppose stack traces in the logs should be enough.